source/cItem.cpp

```diff
@@ -2683,8 +2683,8 @@ void CItem::SendToSocket( CSocket *mSock, [[maybe_unused]] bool drawGamePlayer )
 			auto iVisible = GetVisible();
 			auto ownerObj = GetOwnerObj();
 			if(( iVisible != VT_VISIBLE && iVisible != VT_TEMPHIDDEN )
-				|| iVisible == VT_TEMPHIDDEN &&
-				( !ValidateObject( ownerObj ) || ( ValidateObject( ownerObj ) && ownerObj->GetSerial() != mChar->GetSerial() )))
+				|| ((iVisible == VT_TEMPHIDDEN &&
```

Comment:
This, as well, changes the intended logic. It's supposed to be:
- if visible state is VT_TEMPHIDDEN AND either ownerObj exists OR ownerObj exists and owner serial differs from mChar's serial

Reply:
Does commit cf28051 now contain the correct logic?

Reply:
Not quite 🤔The idea is that the checks on ownerObj & the serial comparison only runs if iVisible equals VT_TEMPHIDDEN. So, maybe... this?
```
if(( iVisible != VT_VISIBLE && iVisible != VT_TEMPHIDDEN )
	|| ( iVisible == VT_TEMPHIDDEN &&
	( !ValidateObject( ownerObj ) || ( ValidateObject( ownerObj ) && ownerObj->GetSerial() != mChar->GetSerial() ))))```

Reply:
Fixed in f4e7e35.